portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,24 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&family=IBM+Plex+Sans&display=swap');
+
+:root, body, html {
+  font-size: 16px;
+  font-family: 'IBM Plex Sans', sans-serif;
+}
+
+header, footer {
+  display: flex;
+  flex-flow: column nowrap;
+  align-items: center;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  max-width: 100vh;
```

Comment:
This seems a bit odd to me. Did you mean 100vw? If not, tall but narrow screens (e.g. someone with their monitor sideways) could look wonky.

Reply:
Good catch! Fixed that.